### PR TITLE
fix:text overflow

### PR DESCRIPTION
**Pull request by KcPele**
Issue
Screenshot from 2024-06-25 00-08-10

fix:
Screenshot from 2024-06-25 08-43-27

### CODE REVIEW

File: packages/nextjs/components/Processes/BundlingProcess/Steps/AssetSelectionStep/AutoDetectedAssets/AutoDetectedAssetItem.tsx [hunk 1]
```diff
@@ -57,21 +55,29 @@ export const AutoDetectedAssetItem = ({ onClick, isSelected, tx, isLoading, imag
       initial={{ opacity: 0 }}
       animate={{ opacity: 1 }}
       onClick={onClick}
-      className={`${isSelected ? "bg-base-200" : ""} ${styles.assetItem}  ${isLoading ? styles.loading : ""}`}
+      className={`${isSelected ? "bg-base-200" : ""} ${styles.assetItem}  ${false ? styles.loading : ""}`}
```
Comment (escottalexander) [inline packages/nextjs/components/Processes/BundlingProcess/Steps/AssetSelectionStep/AutoDetectedAssets/AutoDetectedAssetItem.tsx:58]: Was replacing `isLoading` with `false` intentional?

**Review by escottalexander — CHANGES_REQUESTED** [review]
Need to look into this more but wondering one thing.

File: packages/nextjs/components/Processes/BundlingProcess/Steps/AssetSelectionStep/AutoDetectedAssets/AutoDetectedAssetItem.tsx (hunk 1 above)
Comment (KcPele) [reply packages/nextjs/components/Processes/BundlingProcess/Steps/AssetSelectionStep/AutoDetectedAssets/AutoDetectedAssetItem.tsx:58]: it was a mistake, will have that fixed

**Review by escottalexander — APPROVED** [review]
This works great. Thanks!

**Pull request merged**